examples/eks-cluster-with-observability/main.tf

```diff
@@ -0,0 +1,174 @@
+  }
+
+  private_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/internal-elb"             = "1"
+  }
+
+}
+
+module "aws-eks-accelerator-for-terraform" {
+  source = "../.."
+
+  tenant            = local.tenant
+  environment       = local.environment
+  zone              = local.zone
+  terraform_version = local.terraform_version
+
+  # EKS Cluster VPC and Subnet mandatory config
+  vpc_id             = module.aws_vpc.vpc_id
+  private_subnet_ids = module.aws_vpc.private_subnets
+
+  # EKS CONTROL PLANE VARIABLES
+  create_eks         = true
+  kubernetes_version = local.kubernetes_version
+
+  # EKS MANAGED NODE GROUPS
+  managed_node_groups = {
+    mg_4 = {
+      node_group_name = "managed-ondemand"
+      instance_types  = ["m5.xlarge"]
+      min_size        = 3
+      subnet_ids      = module.aws_vpc.private_subnets
+    }
+  }
+
+  # Enable Amazon Prometheus - Creates a new Workspace id
+  enable_amazon_prometheus = true
+}
+
+resource "aws_iam_policy" "fluentbit-opensearch-access" {
+  name        = "fluentbit-opensearch-access"
+  description = "IAM policy to allow Fluentbit access to OpenSearch"
+  policy      = data.aws_iam_policy_document.fluentbit-opensearch-access.json
+}
+
+module "kubernetes-addons" {
+  source         = "../../modules/kubernetes-addons"
+  eks_cluster_id = module.aws-eks-accelerator-for-terraform.eks_cluster_id
+
+  #K8s Add-ons
+  enable_metrics_server     = true
+  enable_cluster_autoscaler = true
+  enable_argocd             = true
+
+  # Fluentbit
+  enable_aws_for_fluentbit        = true
+  aws_for_fluentbit_irsa_policies = [aws_iam_policy.fluentbit-opensearch-access.arn]
+  aws_for_fluentbit_helm_config = {
+    aws_for_fluentbit_cw_log_group_name = "/${module.aws-eks-accelerator-for-terraform.eks_cluster_id
+    }/worker-fluentbit-logs"
+    values = [templatefile("${path.module}/helm_values/aws-for-fluentbit-values.yaml", {
+      aws_region = data.aws_region.current.name,
+      log_group_name = "/${module.aws-eks-accelerator-for-terraform.eks_cluster_id
+      }/worker-fluentbit-logs",
+      service_account_name = "aws=for-fluent-bit-sa",
+      host                 = aws_elasticsearch_domain.opensearch.endpoint,
+      match                = "*",
+    })]
+  }
+
+  #---------------------------------------
+  # PROMETHEUS and Amazon Prometheus Config
+  #---------------------------------------
+  # Amazon Prometheus Configuration to integrate with Prometheus Server Add-on
+  enable_amazon_prometheus             = true
+  amazon_prometheus_workspace_endpoint = module.aws-eks-accelerator-for-terraform.amazon_prometheus_workspace_endpoint
+
+  #---------------------------------------
+  # COMMUNITY PROMETHEUS ENABLE
+  #---------------------------------------
+  enable_prometheus = true
+  # Optional Map value
+  prometheus_helm_config = {
+    name       = "prometheus"                                         # (Required) Release name.
```

Comment:
these are default values yea? If so, no need to supply from top level module.

Reply:
You're right - I will remove this.